doc/develop/west/manifest.rst

```diff
@@ -980,6 +983,45 @@ is inactive.
 The final group filter specified by :file:`parent/west.yml` and the
 ``manifest.group-filter`` configuration option is ``[+unstable,-optional]``.
 
+
+Filtering projects using a custom manifest
+==========================================
+
+If you want to ``west update`` an existing project, but you don't need all of
+its sub-projects, for example, you want to work on Zephyr, but you don't need
+all the HALs, you can create your own manifest file, which imports the existing
+projects. You can then use the regular filtering features (``name-blocklist``,
+``name-allowlist``, etc.).
+
+Create a west.yml file in a directory outside the other project
+
+.. code-block:: shell
+
+   west config manifest.path <path to new west.yml>
+
+In the new west.yml, import the other project (in this example, Zephyr):
+
+.. code-block:: yaml
+
+   manifest:
+     defaults:
+       remote: upstream
+     remotes:
+       - name: upstream
+         url-base: https://github.com/zephyrproject-rtos
+     projects:
+       - name: zephyr
+         revision: HEAD~0
+         import:
+           name-blocklist:
+             # List all the projects you want to exclude
+             - hal_*
+             - etc...
```

Comment:
```
  projects:
    - name: sdk-nrf
      path: nrf
      remote: ncs
      revision: d579503ee9f19139cfd15dcaabe9b5f5be1e2d8a
      import: true
        name-allowlist:
          - hal_nordic
```

There seems to be a problem with the schema.
Is there a minimum version required for this feature?

Reply:
That looks like invalid YAML

`import: true` makes `import` a bool, while on the next line, `name-allowlist:` tries to make it a dict. Have you tried removing ` true`?

Reply:
That worked, thanks!